src/Taps.cpp

```diff
@@ -76,4 +83,48 @@ std::unique_ptr<InputStream> Tap::instantiate(const Configurable *config, const
     return module;
 }
 
+bool Tap::tags_validate_yaml(const YAML::Node &tag_yaml, bool all)
+{
+    bool any_match = false;
+    for (YAML::const_iterator it = tag_yaml.begin(); it != tag_yaml.end(); ++it) {
+        if (!it->second.IsScalar()) {
+            throw PolicyException(fmt::format("tag key '{}' must have scalar value", it->first));
```

Comment:
think we probably need a TapException and not PolicyException because tags are set at pktvisor config load time, not policy

Reply:
agreed

Reply:
done